extensions/standard-processors/processors/HashContent.h

```diff
@@ -38,21 +38,13 @@
 #include "core/ProcessSession.h"
 #include "core/Resource.h"
 #include "io/BaseStream.h"
+#include "utils/StringUtils.h"
 
 using HashReturnType = std::pair<std::string, int64_t>;
 
 namespace {
 #define HASH_BUFFER_SIZE 16384
 
-  std::string digestToString(const unsigned char * const digest, size_t size) {
```

Comment:
is there validation to ensure this produces the exact same expected output?

Reply:
Yes, both in exizsting known answer tests in https://github.com/apache/nifi-minifi-cpp/blob/master/extensions/standard-processors/tests/unit/HashContentTest.cpp#L50 and there are unit tests for uppercase hexencoding in this PR.

Reply:
Thank you!